apps/frontend/src/extensions/SlashCommand/SlashCommand.ts

```diff
@@ -1,4 +1,5 @@
-import { Editor, Extension } from "@tiptap/core"
+import { Editor } from "@tiptap/react"
+import { Extension } from "@tiptap/core"
 import { PluginKey } from "@tiptap/pm/state"
 import { ReactRenderer } from "@tiptap/react"
```

Comment:
imported multiple times.

Reply:
@sajdakabir it needs to be imported multiple times because editor needs contentComponent variable which is present is @tiptap/react but not in /core and the extension needs 
command which is in @tiptap/core.